### ISSUE TITLE
[Suggestion] Auto-report suspicious or miscategorized file types.

### ISSUE THREAD

**Issue by such-doge** [nyaadevs/nyaa#216]
It'd be a good idea to automatically flag new torrents that contain file types that don't really belong in their particular category. A moderator could then review the torrent. I know this isn't a total solution, but it's a way to catch low-hanging fruit without relying on users. More importantly, the torrent can be fixed or deleted before the front page cache catches up to the database.

Examples:

* Flag .scr files universally. This one could benefit from shadow-banning too.
* Flag .exe files in non-software torrents.
* Flag .rar/.zip files in video torrents.
* Flag video torrents that contain no common video extensions (.rmvb, .webm?, .wmv, .mp4, .avi, .ogv, .mkv, .ass for subtitle-only releases, mpeg extensions, plus the various extensions used in DVD and BD rips).
* Flag audio torrents that contain video. This is useful for catching miscategorized music videos.
* Exempt trusted users from auto-flagging. This gets around the issue of trusted fansub groups who like to include xdelta3.exe.

**Comment by ReimuHakurei** [comment]
flagging rar/zip files in torrents is a bad idea, i think, since many groups include things like artwork in that form.

however, a video torrent that say... ONLY contains that could be suspicious.

**Comment by such-doge** [comment]
That's true. And video torrents with only those files would be covered by the next condition in the list.

**Comment by Arylide** [comment]
Ideally such a system would be highly configurable to decide what parameters are enforced on what categories, etc. If anyone wanted to work on this.

**Comment by TheAMM** [comment]
I have a proto of #351 but it's far from user(mod) friendly.
